### ISSUE TITLE
ISO-TP with CANFD

### ISSUE THREAD

**Issue by trollkno1l** [tonton81/FlexCAN_T4#63]
I've read a lot about your FlexCAN-library and surprised about the range of functionalities.

I would like to use ISO-TP on CANFD, but after some testing and viewing the examples I have the concern that ISO-TP with CANFD is not supported... maybe im wrong.

**Comment by tonton81** [comment]
yes unfortunately this ISOTP library was designed for CAN2.0 mode based on 8 byte standard fixed frame layout, in it's current state it would unlikely be capable of running with FD payloads without a major rewrite, and obviously larger buffer sizes & ram to re-assemble the payload
